### DISCUSSION TITLE
[1.3] 2022-02-26

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#190] [General]
# Wired
* Add support for Virtual Boy
* Add support for VirtualTap Mode & Palette buttons
* Add support for SEA board
* Add support for open drain parallel output
* Fixes Jaguar Breakout 2000 issue

# Bluetooth
* Increase BLE devices maximum from 3 to 8

# General
* Update to esp-idf 5.0.0
* Increase SPIFFS partition to 960KB

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.3'>[1.3] 2022-02-26</a>.</em>

**Comment by pmgducati** [comment]
Does this release include the update for the PCE Controls?

https://github.com/darthcloud/BlueRetro/issues/168

**Reply by darthcloud** [reply]
No this will be in next one sorry.
